lib/src/realm_class.dart

```diff
@@ -496,12 +490,6 @@ class Realm implements Finalizable {
     }
   }
 
-  void _ensureWritable() {
```

Comment:
I am not sure I am following. Has this moved to core or why we are removing this from here?

Reply:
Yes, this check is done in core and the exception is thrown by core. We don't need to it here. It doesn't cover all the cases that core checks.